eo-runtime/src/test/eo/org/eolang/gray/rho-of-add-should-not-change.eo

```diff
@@ -0,0 +1,15 @@
++package org.eolang
```

Comment:
@dours see other `EO`, ​​they all have **licenses** at the top. I suggest including this here too.
Tks